getting_started/workflow/project_setup/project_organization.rst

```diff
@@ -53,5 +54,18 @@ resulted in an organization hassle for most developers.
 Because of this, assets are now transparently imported from within the project
 folder.
 
-If a folder shouldn't be imported into Godot, an exception can be made with a
-.gdignore file.
+Ignoring specific folders
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To prevent Godot from importing files contained in a specific folder, create
+an empty file called ``.gdignore`` in the folder (the leading ``.`` is required).
+This can be useful to speed up the initial project importing.
```

Comment:
consider expanding this to the whole `rationale` sub-section which may mention not only import speedup, but also making a room for project-related but not godot-related stuff like docs, raw art files, etc.